lib/landing_page.dart

```diff
@@ -0,0 +1,449 @@
+                  style: TextStyle(
+                    fontSize: 14,
+                    fontWeight: FontWeight.w200,
+                    color: Colors.white,
+                  )),
+              SizedBox(height: 110),
+              Text('Kontaktiere uns gerne, falls du weitere Fragen hast.',
+                  textAlign: TextAlign.center,
+                  style: TextStyle(
+                    fontSize: 24,
+                    fontFamily: 'Impact',
+                    color: Colors.white,
+                  )),
+
+              SizedBox(height: 30),
+              Text(' info@places.rocks ',
+                  textAlign: TextAlign.center,
+                  style: TextStyle(
+                    fontSize: 14,
+                    fontWeight: FontWeight.w200,
+                    color: Colors.white,
+                  )),
+              SizedBox(height: 80),
+              Align(
+                alignment: Alignment.center,
+                child: RaisedButton(
+                  color: Colors.white10,
+                  onPressed: _launchDSE,
+                  child: Text(
+                    'Impressum',
+                    style: TextStyle(
+                      fontSize: 14,
+                      color: Colors.grey,
+                    ),
+                  ),
+                ),
+              ),
+              Align(
+                alignment: Alignment.center,
+                child: RaisedButton(
+                  color: Colors.white10,
+                  onPressed: _launchURL,
+                  child: Text(
+                    'Datenschutz',
+                    style: TextStyle(
+                      fontSize: 14,
+                      color: Colors.grey,
+                    ),
+                  ),
+                ),
+              ),
+              SizedBox(height: 80),
+              Text("Places Rocks UG - all rights reserved",
+                  textAlign: TextAlign.center,
+                  style: TextStyle(
+                    fontSize: 14,
+                    fontWeight: FontWeight.w200,
+                    color: Colors.grey,
+                  )),
+              SizedBox(height: 30),
+            ],
+          ),
+        ),
+      ],
+    );
+  }
+}
+
+class RandomImage extends StatefulWidget {
+  @override
+  _RandomImageState createState() => _RandomImageState();
+}
+
+class _RandomImageState extends State<RandomImage> {
+  @override
+  Widget build(BuildContext content) {
+    Random random = new Random();
+    int randomNumber = random.nextInt(4) + 1;
+    return Container(
+        width: 280,
+        child: Image.asset('assets/images/subtitle/$randomNumber.png'));
+  }
+}
+
+_launchURL() async {
+  const url = 'https://places.rocks/monsterjump-dse';
+  if (await canLaunch(url)) {
+    await launch(url);
+  } else {
+    throw 'Could not launch $url';
+  }
+}
+
+_launchDSE() async {
+  const url = 'https://places.rocks/impressum';
+  if (await canLaunch(url)) {
+    await launch(url);
```

Comment:
Was this tested on web? Isn't there a better way to do this on Flutter web?

Reply:
yes, and indeed